spring-integration-core/src/test/java/org/springframework/integration/handler/LoggingHandlerTests.java

```diff
@@ -16,35 +16,31 @@
 
 package org.springframework.integration.handler;
 
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.fail;
-import static org.mockito.Mockito.mock;
-import static org.mockito.Mockito.never;
-import static org.mockito.Mockito.spy;
-import static org.mockito.Mockito.times;
-import static org.mockito.Mockito.verify;
-import static org.mockito.Mockito.when;
+import static org.junit.Assert.*;
+import static org.mockito.Mockito.*;
```

Comment:
@artembilan fixed. BTW I've used standard [IntelliJ IDEA code format settings](https://github.com/spring-projects/spring-framework/tree/master/src/idea). Them should be fixed then. I'll provide needed changes.